test/StdAssertions.t.sol

```diff
@@ -920,8 +920,8 @@ contract TestTest is Test {
         address targetB,
         bytes memory callDataA,
         bytes memory callDataB,
-        bytes memory returnDataA,
-        bytes memory returnDataB,
+        bytes memory,
+        bytes memory,
```

Comment:
Should we just remove these two arguments instead? cc @0xPhaze

Reply:
From the way it looks, they can be removed. I think I had originally used it to compare return/revert data, but saw that the pattern with `vm.expectEmit` and `expectFail` covered it. In that case, I would maybe just nit for the parameter order to be conformed with the actual `assertEqCall` parameter order.

```solidity
    function _assertEqCall(
        address targetA,
        address targetB,
        bytes memory callDataA,
        bytes memory callDataB,
        bytes memory returnDataA,
        bytes memory returnDataB,
        bool strictRevertData,
        bool expectFail
        )
```

to

```solidity
    function _assertEqCall(
        address targetA,
        bytes memory callDataA,
        address targetB,
        bytes memory callDataB,
        bool strictRevertData,
        bool expectFail
        )
```

Reply:
ok I removed them!